plugins/lang-groovy/src/test/java/org/elasticsearch/messy/tests/TDigestPercentilesTests.java

```diff
@@ -383,22 +364,6 @@ public void testScriptMultiValued() throws Exception {
     }
 
     @Override
-    public void testScriptExplicitMultiValued() throws Exception {
```

Comment:
Sounds reasonable. This was also the only thing I would have mentioned. Otherwise LGTM.